### ISSUE TITLE
--no-guide not working. Checkov still tries to access www.bridgecrew.io

### ISSUE THREAD

**Issue by steffencircle** [bridgecrewio/checkov#1609]
We want to run checkov in a completely offline manner as part of our Terraform Pipeline.

So in order to disable any online checks we run checkov using to "--no-guide" parameter.

However, when we set the LOG_LEVEL to debug , we can still see that checkov tries to access www.bridgecrew.io/api/v1/guidelines via our Proxy.

The container has the "https_proxy" environment variable set for Terraform to work, but as said initially we do not want the checkov online checks.

One additional note, our proxy requires authentication and the https_proxy env var contains the correct username/password, however checkov although seems to be not able to use that.

Here is an error log:

2021-09-14 09:01:24,682 [MainThread  ] [DEBUG]  Failed to get the guidelines from https://www.bridgecrew.cloud/api/v1/guidelines, error:
HTTPSConnectionPool(host='www.bridgecrew.cloud', port=443): Max retries exceeded with url: /api/v1/guidelines (Caused by ProxyError('Cannot connect to proxy.', OSError('Tunnel connection failed: 407 Proxy Authentication Required')))

 - OS: Linux (Docker)
 - Checkov Version 2.0.405

**Comment by mikeurbanski1** [comment]
Hi @steffencircle , we will look into this; in the meantime you can set the environment variable `BC_SKIP_MAPPING=TRUE`

**Comment by njgibbon** [comment]
Hi all,

I'm unable to replicate.

If I just download checkov, disconnect my computer from the internet and run checkov against some arbitrary terraform module it runs fine without guidelines.

```
pip install checkov --upgrade
checkov --version
# 2.0.484
cd <some/terraform/module>
LOG_LEVEL=DEBUG checkov --no-guide --directory .
```

**Comment by nimrodkor** [comment]
Fixed in #1757
